practice-one/src/assets/css/footer.css

```diff
@@ -0,0 +1,60 @@
+.footer-section {
+  background-color: var(--black-color);
+  padding-top: 105px;
+  padding-left: 147px;
+  padding-right: 151px;
+  padding-bottom: 91px;
+}
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-item-top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+.footer-item-bottom {
+  display: flex;
```

Comment:
duplicate code

Reply:
I fixed it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/1dc66147-b5a2-4e27-b7d2-5f0882c7dbf5)